docs/ecosystem/index.md

```diff
@@ -4,5 +4,8 @@
 import DocCardList from '@theme/DocCardList';
 import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 
+![ecosystem map](./img/2.jpeg)
+Note the ecosystem map above is as of July 26 2022.
```

Comment:
Maybe add this text:
Astar Network is a permissionless blockchain, only the most used dApps are listed in the overview. (source: DappRadar)

Reply:
Updated the text! Thank @fiexer